### PR TITLE
Multi select delete

### PR DESCRIPTION
**Pull request by Hicham010**
I added the multi-select delete feature.

### CODE REVIEW

**Review by Almightywesley — APPROVED** [review]

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java [hunk 1]
```diff
@@ -44,7 +55,25 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
+        adapter = new MemoriesAdapter(items, new MemoriesAdapter.OnClickListener() {
+            @Override
+            public void onItemClick(View view, Memory obj, int pos) {
+                if (adapter.getSelectedItemCount() > 0) {
+                    enableActionMode(pos);
+                } else {
+                    Intent intent = new Intent(getContext(), MemoryDetailActivity.class);
+                    intent.putExtra("EXTRA_SESSION_ID", obj.getId());
+                    view.getContext().startActivity(intent);
+                    AnalyticsUtil.selectContent(getContext(), "SearchOrList");
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java]: Please replace `"SearchOrList"` with `"List"`

It was always in the adapter, so we couldn't separate it at first. Now we can.

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java [hunk 2]
```diff
@@ -39,7 +40,19 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
 
         memories = Database.getInstance().getMemories();
-        adapter = new MemoriesAdapter(memories);
+        adapter = new MemoriesAdapter(memories, new MemoriesAdapter.OnClickListener() {
+            @Override
+            public void onItemClick(View view, Memory obj, int pos) {
+                Intent intent = new Intent(getContext(), MemoryDetailActivity.class);
+                intent.putExtra("EXTRA_SESSION_ID", obj.getId());
+                view.getContext().startActivity(intent);
+                AnalyticsUtil.selectContent(getContext(), "SearchOrList");
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java]: Please replace `"SearchOrList"` with `"Search"` here.

It was always in the adapter, so we couldn't separate it at first. Now we can.

File: app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -54,6 +55,10 @@
     <string name="action_skip">Overslaan</string>
     <string name="action_restart_tutorial">Herstart Uitleg</string>
 
+    <!-- snackbar -->
+    <string name="snackbar_description">Weet u zeker dat u het wilt verwijderen?</string>
+    <string name="snackbar_undo">Ongd. maken</string>
```
Comment (MrTheGood) [inline app/src/main/res/values-nl/strings.xml]: Change this to 'Zet Terug'

**Review by MrTheGood — APPROVED** [review]
I have some minor change requests.

**Nice work though!**

**Review by MrTheGood — CHANGES_REQUESTED** [review]
see comment above. Accidentally approved.

File: app/src/main/res/values-nl/strings.xml (hunk 3 above)
Comment (Hicham010) [reply app/src/main/res/values-nl/strings.xml]: Ik heb "Onged. Maken" gezet omdat het ook bij gmail staat als je een mail weg haalt. Dus ik ga het zo laten.

**Comment by Hicham010** [comment]
I also added some small improvements, like a toast for the empty filter and hide the double toolbar so only the delete toolbar is present while deleting

**Review by MrTheGood — APPROVED** [review]

**Comment by MrTheGood** [comment]
Nice work!

**Pull request merged**